SVGImageView.m

```diff
@@ -43,8 +43,8 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder
 - (instancetype)initWithFrame:(CGRect)frame
 {
     if ((self = [super initWithFrame:frame])) {
-        self.wantsLayer = YES;
```

Comment:
~Why reverse this?  Also, it's not reversed in initWithCoder:~

This also needs to be reversed in initWithCoder: